include/mbedtls/pk.h

```diff
@@ -772,6 +772,7 @@ static inline mbedtls_rsa_context *mbedtls_pk_rsa(const mbedtls_pk_context pk)
 {
     switch (mbedtls_pk_get_type(&pk)) {
         case MBEDTLS_PK_RSA:
+        case MBEDTLS_PK_RSA_ALT:
```

Comment:
This is not correct. `PK_RSA_ALT` contexts can contain anything as their `pk_ctx` pointer, so it's only safe to cast to `mbedtls_rsa_context *` when the type is `PK_RSA` exactly - as reflect [in this function's documentation](https://github.com/Mbed-TLS/mbedtls/blob/development/include/mbedtls/pk.h#L756).